CIP-XXXX/README.md

```diff
@@ -0,0 +1,113 @@
+---
+CIP:
+  ?
+Title: Cardano URIs - CIP-0019 Address Representation
+Category: Wallets
+Status: Proposed
+Authors:
+  - Steven Johnson <steven.johnson@iohk.io>
+Implementors: [ ]
+Discussions:
+  - https://github.com/cardano-foundation/CIPs/pull/?
+Created: 2024-08-23
+License: CC-BY-4.0
+---
+
+## Abstract
+
+This CIP proposes an extension to [CIP-13] to allow easy and unambiguous encoding of
+[CIP-0019]/[CIP-0105] Addresses into URL's.
+
+## Motivation: why is this CIP necessary?
+
+[CIP-0013] defines encoding for payment addresses, however [CIP-0019] and [CIP-0105] define
+numerous other address types.
+
+These addresses cannot currently be encoded into URLs unambiguously.
+This extension proposes a simple and forward compatible method of encoding such addresses into the URL scheme defined by [CIP-0013].
+
+[x509] certificates can contain name or alternative name information related to either the issuer of
```

Comment:
Maybe add c509 certificates too

Reply:
I just refer to x509 in the general sense.  c509 just being a different way to encode the cert.